utils/python/CIME/XML/entry_id.py

```diff
@@ -106,28 +145,55 @@ def get_value(self, vid, attribute={}, resolved=True, subgroup=None):
         type_str = self._get_type_info(node)
         return convert_to_type(val, type_str, vid)
 
-    def get_values(self, vid, att, resolved=True):
+    def get_values(self, item, attribute={}, resolved=True, subgroup=None): # (self, vid, att, resolved=True , subgroup=None ):
```

Comment:
I think that your get_values is different enough from the get values in #130 that maybe they should be seperate functions?